src/test/java/AcmeShop.java

```diff
@@ -114,10 +107,105 @@ public void addingToCart() {
             builder4.moveToElement(fieldComments, width, height).clickAndHold().moveByOffset(0, height*5)
```

Comment:
No need to create a new builder for every complex action within one method use the builder you've instantiated before

Reply:
Fixed